changelog.txt

```diff
@@ -8,8 +8,10 @@
  * Added new Dynamic API using DynamicRealm and DynamicRealmObject.
  * Added Realm.getSchema() and DynamicRealm.getSchema().
  * Realm.createOrUpdateObjectFromJson() now works correctly if the RealmObject class contains a primary key (#1777).
- * Updated Realm Core to 0.95.1
  * Realm.compactRealm() doesn't throw an exception if the Realm file is opened. It just returns false instead.
+ * Updated Realm Core to 0.95.3
+   - Fixed a bug where RealmQuey.average(String) returned a wrong value for a nullable Long/Integer/Short/Byte field (#1803).
```

Comment:
```
    public double average(String fieldName) {
        long columnIndex = schema.getFieldIndex(fieldName);
        switch (table.getColumnType(columnIndex)) {
            case INTEGER:
                return query.averageInt(columnIndex);
            case DOUBLE:
                return query.averageDouble(columnIndex);
            case FLOAT:
                return query.averageFloat(columnIndex);
            default:
                throw new IllegalArgumentException(String.format(TYPE_MISMATCH, fieldName, "int, float or double"));
        }
    }
```

Apparently we do support it for byte, I don't have any sensible use case for it, but no need to change anything in this PR then